Task 9/script.js

```diff
@@ -6,3 +6,18 @@ title, director, budget
 Metodas: 
 wasExpensive() - jeigu filmo budget bus didesnę nei 100 000 000 mln USD, tada gražins true, kiru atveju false 
 ------------------------------------------------------------------------------------------------------ */
+class Movie {
+  constructor(title, director, budget) {
+    this.title = title;
+    this.director = director;
+    this.budget = budget;
+  }
+  wasExpensive() {
+    return this.budget > 100000000 ? true : false
```

Comment:
Uztektu tiesiog taip
```suggestion
    return this.budget > 100000000;
```

nes tas palyginimas jau grazina true arba false reiksme.